content/tutorials/webrtc/infrastructure/en/index.html

```diff
@@ -140,20 +131,20 @@ <h3 id="rtcpeerconnection-signaling-offer-answer-and-candidate">RTCPeerConnectio
   <p><a href="http://www.urbandictionary.com/define.php?term=strewth">Strewth!</a></p>
 </blockquote>
 
-<p>Alice and Eve also need to exchange network information. The expression 'finding candidates' refers to the process of finding network interfaces and ports using the <a href="http://en.wikipedia.org/wiki/Interactive_Connectivity_Establishment">ICE framework</a>.</p>
+<p>Alice and Eve also need to exchange network information. The expression "finding candidates" refers to the process of finding network interfaces and ports using the <a href="http://en.wikipedia.org/wiki/Interactive_Connectivity_Establishment">ICE framework</a>.</p>
 
 <ol style="list-style-type: decimal">
-  <li>Alice creates an RTCPeerConnection object with an <em>onicecandidate</em> handler.</li>
+  <li>Alice creates an <code>RTCPeerConnection</code> object with an <code>onicecandidate</code> handler.</li>
   <li>The handler is called when network candidates become available.</li>
-  <li>In the handler, Alice sends stringified candidate data to Eve, via their signaling channel.</li>
-  <li>When Eve gets a candidate message from Alice, she calls <em>addIceCandidate()</em>, to add the candidate to the remote peer description.</li>
+  <li>In the handler, Alice sends stringified candidate data to Eve through their signaling channel.</li>
+  <li>When Eve gets a candidate message from Alice, she calls <code>addIceCandidate()</code> to add the candidate to the remote peer description.</li>
 </ol>
 
 <p>JSEP supports <a href="http://tools.ietf.org/html/draft-ietf-rtcweb-jsep-03#section-3.4.1">ICE Candidate Trickling</a>, which allows the caller to incrementally provide candidates to the callee after the initial offer, and for the callee to begin acting on the call and setting up a connection without waiting for all candidates to arrive.</p>
 
 <h3 id="coding-webrtc-for-signaling">Coding WebRTC for signaling</h3>
 
-<p>Below is a <a href="https://w3c.github.io/webrtc-pc/#simple-peer-to-peer-example">W3C code example</a> that summarizes the complete signaling process. The code assumes the existence of some signaling mechanism, <em>SignalingChannel</em>. Signaling is discussed in greater detail below.</p>
+<p>The following code snippet is a <a href="https://w3c.github.io/webrtc-pc/#simple-peer-to-peer-example">W3C code example</a> that summarizes the complete signaling process. The code assumes the existence of some signaling mechanism&mdash;<code>SignalingChannel</code>. Signaling is discussed in greater detail later.</p>
```

Comment:
Maybe reinstate the comma rather than the `&mdash;`?

(`SignalingChannel` is a variable name in the code.)

Reply:
Will fix

Reply:
Will fix